qa/1991.out

```diff
@@ -0,0 +1,289 @@
+QA output created by 1991
+
+pcp-netstat output : Display default output
+05:24:46
+
+05:24:47
+
```

Comment:
@mohith-kumar-thummaluru is the above expected?  (seems to be no output?)

Reply:
hi scott, by default with no options provided as input, this tool prints no data. This is expected output

Reply:
> 

Is there a reason?  It would be very unexpected for users I think ... if there is some useful default option that can displayed, we should lead with that and allow fine-tuning of the display through other options.  But not displaying anything doesn't really make sense to me.

Reply:
ok, I agree with you. I'll add a useful output for the default option and update you